src/sagas/session.js

```diff
@@ -84,9 +92,16 @@ export function* setupSession(
     const { authType } = auth;
     if (
       authType != "anonymous" &&
-      (!userId || !userId.startsWith(authType + ":"))
+      (!userId ||
+        (!userId.startsWith(authType + ":") &&
+          // If the authType is openid, the userId doesn't start with "openid:" but "auth0:".
+          (authType.startsWith("openid-") && !userId.startsWith("auth0:"))))
```

Comment:
I don't understand this test.

The user id starts with the provider name (the right part of authType if I understand correctly)

Reply:
are you sure about that? In my tests, i got `auth0:...` back, which was a surprise to me!

Reply:
seems my issue with the `auth0:` prefix was due to some configuration problem for the kinto running locally... I've fixed that (and the test)